cli/cmd/get_cmd.go

```diff
@@ -29,7 +29,7 @@ var getCmd = &cobra.Command{
 			return "", err
 		}
 
-		if output == string(formatters.JSON) {
+		if output == string(formatters.JSON) || output == string(formatters.Pretty) {
```

Comment:
This was done to solve #2760, which was not showing values because we were not returning augmented values on get for pretty output too.